scripts/update_voice_data.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python3
+
+# Update the voice data
+import asyncio
+import subprocess
+from pathlib import Path
+
+import aiohttp
+
+voice_data_path = Path(__file__).parent.parent / "hass_nabucasa/voice_data.py"
+
+REGION = "eastus"
+LIST_VOICES_URL = (
+    f"https://{REGION}.tts.speech.microsoft.com/cognitiveservices/voices/list"
```

Comment:
Do the different regions have different voices?
Would this need to be mapped as well?